src/Table.js

```diff
@@ -15,7 +15,7 @@ const TableBody = props => {
 const Table = (props) => {
   const { messagesData, removeMessage } = props;
   return (
-    <TableBody messagesData={messagesData} removeMessage={removeMessage} />  
+    <TableBody messagesData={messagesData} key={props.index} removeMessage={removeMessage} />  
```

Comment:
key={props.index} this prop is not handled in the TableBody, so we don't need it here